core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -908,10 +974,18 @@ public void completeFile(AlluxioURI path, CompleteFileOptions options)
       InvalidFileSizeException, FileAlreadyCompletedException, AccessControlException {
     Metrics.COMPLETE_FILE_OPS.inc();
     try (JournalContext journalContext = createJournalContext();
-        LockedInodePath inodePath = mInodeTree.lockFullInodePath(path, InodeTree.LockMode.WRITE)) {
-      mPermissionChecker.checkPermission(Mode.Bits.WRITE, inodePath);
+         LockedInodePath inodePath = mInodeTree.lockFullInodePath(path, InodeTree.LockMode.WRITE);
+         MasterAuditContext auditContext = createAuditContext("completeFile", path, null,
+             inodePath.getInode())) {
```

Comment:
In `completeFile` method, I think the expected behavior when the target inode does not exist is to throw a FileDoesNotExistException? Correct me if I'm wrong.